lib/hyrax/specs/shared_specs/metadata.rb

```diff
@@ -92,3 +92,70 @@
       .to contain_exactly('title')
   end
 end
+
+RSpec.shared_examples 'a model with collection basic metadata' do
+  subject(:resource) { described_class.new }
+  let(:date)         { Time.zone.today }
```

Comment:
Including this in shared_specs can cause apps that override basic metadata to fail this set of specs.  I'll look at how works handle basic metadata.  I would think they would have the same issue.